.github/workflows/test-sigstore-tuf-repo.yml

```diff
@@ -0,0 +1,48 @@
+name: test-tuf-sigstore-repo
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+  workflow_dispatch:
+
+jobs:
+  run-tests:
+    name: Test TUF client with sigstore TUF repo
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - 14.17.0
+          - 14.x
+          - 16.13.0
+          - 16.x
+          - 18.0.0
+          - 18.x
+        platform:
+          - os: ubuntu-latest
+            shell: bash
+          - os: macos-latest
+            shell: bash
+          - os: windows-latest
+            shell: cmd
+    runs-on: ${{ matrix.platform.os }}
+    defaults:
+      run:
+        shell: ${{ matrix.platform.shell }}
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3
+      - name: Setup node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+      - name: Run TUF client
+        uses: ejahnGithub/tuf-client-fetch-target-action@v1.1.8
```

Comment:
Using an action is neat but wondering if it would be more dev-friendly to rely on the script you have included and run it here? That way anyone wanting to test this locally can easily replicate this by just running the script on their machine. Is there any reason we can't do this here?

Reply:
yeah, I agree w/ @feelepxyz here. I think this makes more sense as just a local script in this repo.